Ghidra/Features/Base/src/main/java/ghidra/app/util/navigation/GoToQuery.java

```diff
@@ -35,19 +37,27 @@
 import ghidra.framework.plugintool.Plugin;
 import ghidra.framework.plugintool.PluginTool;
 import ghidra.program.model.address.*;
+import ghidra.program.model.listing.Data;
+import ghidra.program.model.listing.Listing;
 import ghidra.program.model.listing.Program;
 import ghidra.program.model.mem.Memory;
 import ghidra.program.model.mem.MemoryBlock;
 import ghidra.program.model.symbol.*;
 import ghidra.program.util.AddressEvaluator;
 import ghidra.program.util.ProgramLocation;
 import ghidra.util.Msg;
+import ghidra.util.NumericUtilities;
 import ghidra.util.SystemUtilities;
 import ghidra.util.table.AddressArrayTableModel;
 import ghidra.util.table.GhidraProgramTableModel;
 import ghidra.util.task.TaskMonitor;
 
 public class GoToQuery {
+	private static final Pattern ARRAY_COMPONENT_PATTERN =
```

Comment:
some human readable javadocs would be appreciated for regexes (especially ones this complex), with example of what it is intended to match.

Reply:
Are those docs sufficient?